### PR TITLE
Add initial draft of wg lts annual report

### LINKED ISSUE #7790: 2023 Annual Report: WG LTS
2023 Annual Report: WG LTS

Chairs: @jeremyrickard @liggitt @micahhausler 
Liaison: @palnabarun

Actions for the chair/organizer of the community group:
- [ ] Consult your community group to complete draft of report
- [ ] If needed, consult with Steering Committee liaison on any private issues or concerns that arrise while completing report
- [ ] Submit PR to git.k8s.io/community repo with copy of report, assigning your Steering Committee liaison for review
- [ ] Steering Committee reviews report, and may make comments/ask questions
- [ ] Once all comments are addressed, Steering Committee will approve report to merge
- [ ] If needed, any follow up items may be brought to the "Chairs, Tech Leads, and Organizers" meeting in April

Once all the above items are complete, this issue may be `/close`'d

Key dates:
- Initial PR to communtiy repo should be opened by May 1, 2024
- PR should be reviewed and merged by May 22, 2024

More detailed information on the annual reports process is available [here](https://git.k8s.io/community/committee-steering/governance/annual-reports.md).

Your group's report template is available here: https://git.k8s.io/community/wg-lts/annual-report-2023.md

If you have any questions or concerns about this process, you may reach the Steering Committee via the following methods:
- Slack/E-mail your liaison directly
- The public #steering-committee channel in Slack
- The public mailing list steering@kubernetes.io
- The private mailing list steering-private@kubernetes.io for any private or sensitive issues.

/assign @jeremyrickard @liggitt @micahhausler 
/wg lts
/committee steering
/area annual-reports

### PR DESCRIPTION
**Pull request by jeremyrickard**
**Which issue(s) this PR fixes**:

Fixes #7790

### CODE REVIEW

File: wg-lts/annual-report-2023.md [hunk 1]
```diff
@@ -5,23 +5,23 @@
...
 1. What work did the WG do this year that should be highlighted?
 
-<!--
-   Some example items that might be worth highlighting:
-   - artifacts
-   - reports
-   - white papers
-   - work not tracked in KEPs
--->
+During 2023, we worked to restart the WG.
```
Comment (palnabarun) [inline wg-lts/annual-report-2023.md:8]: Can you also please add notes from the discussion that happened during KCS EU 23?

File: wg-lts/annual-report-2023.md [hunk 2]
```diff
@@ -5,23 +5,23 @@
...
+
+We also designed and published a [survey](https://bit.ly/k8s-upgrade-survey) to capture pain points and better understand challenges Kubernetes users face with upgrading. We collected 223 survey responses. From this data, we identified some  areas where we can better communicate improvements that have already been made to the project. We plan to work on several of these during 2024 and will present analyzed results from the survey to the community and sponsoring SIGs during 2024. 
+
+We also hosted a [maintainer track session](https://www.youtube.com/watch?v=0fngdOlwZtQ) at KubeCon NA in Chicago to discuss what the working group wanted to achieve.
+
+We have been meeting on a bi-weekly cadence, but have not provided updates to the sponsoring SIGs as most of 2023 was spent forming and drafting the survey. We will provide updates to the sponsoring SIGS during 2024.
```
Comment (palnabarun) [inline wg-lts/annual-report-2023.md:14]: nit,
```suggestion
We have been meeting on a bi-weekly cadence, but have not provided updates to the sponsoring SIGs as most of 2023 was spent forming and drafting the survey. We will provide updates to the sponsoring SIG during 2024.
```

File: wg-lts/annual-report-2023.md (hunk 1 above)
Comment (bridgetkromhout) [reply wg-lts/annual-report-2023.md:8]: Here are the notes for the discussion at KCS EU 2023: https://docs.google.com/document/d/1C7utWMjGpDcBc5TB_ooddERsid59sFLC7gqLzyS5aKY/edit?usp=sharing

File: wg-lts/annual-report-2023.md (hunk 2 above)
Comment (humblec) [reply wg-lts/annual-report-2023.md:14]: the change `SIG` is applicable on line 10 too..

File: wg-lts/annual-report-2023.md [hunk 3]
```diff
@@ -5,23 +5,23 @@
...
-<!--
-   Some example items that might be worth highlighting:
-   - artifacts
-   - reports
-   - white papers
-   - work not tracked in KEPs
--->
+During 2023, we worked to restart the WG.
+
+We also designed and published a [survey](https://bit.ly/k8s-upgrade-survey) to capture pain points and better understand challenges Kubernetes users face with upgrading. We collected 223 survey responses. From this data, we identified some  areas where we can better communicate improvements that have already been made to the project. We plan to work on several of these during 2024 and will present analyzed results from the survey to the community and sponsoring SIGs during 2024.
```
Comment (humblec) [inline wg-lts/annual-report-2023.md:10]: imo, its better to list out the main topics/areas listed over here. Please discard this comment if its not a practice.

File: wg-lts/annual-report-2023.md (hunk 2 above)
Comment (bridgetkromhout) [reply wg-lts/annual-report-2023.md:14]: There are several sponsoring SIGs for this WG.

**Comment by bridgetkromhout** [comment]
2023 talks to list:  

[[KCSNA 2023] Swimming with the current: make it easy to stay up-to-date - Jordan Liggitt](https://www.youtube.com/watch?v=OtigVP3lRh4)

[[KubeCon NA 2023] Community-Powered Kubernetes LTS: Ensuring Stability and Compatibility While Driving Innovation - Jeremy Rickard](https://kccncna2023.sched.com/event/1R4cZ/sponsored-keynote-community-powered-kubernetes-lts-ensuring-stability-and-compatibility-while-driving-innovation-jeremy-rickard-principal-software-engineer-microsoft-azure)

[[KubeCon NA 2023] What's up with Kubernetes Long Term Support? - Jeremy Rickard](https://kccncna2023.sched.com/event/1R2tj/whats-up-with-kubernetes-long-term-support-jeremy-rickard-microsoft)

**Comment by pohly** [comment]
/reopen

Please continue with the annual report... :sweat_smile:

**Pull request closed without merging**